manual/sphinx/_apidoc/boutdata.rst

```diff
@@ -0,0 +1,96 @@
+boutdata package
```

Comment:
Is it possible to auto-generate this file, it looks quite formulaic?

Reply:
These were auto-generated by `sphinx-apidoc`, but then I edited them slightly (moved the full module docs to the top) and committed those edits.

I could add the autogeneration to `conf.py`, similar to the `breathe` docs, but I'm not 100% sure the documentation is laid out the best it could be